pages/guideforms.py

```diff
@@ -38,6 +41,27 @@ def validate(self, value):
         for email in value:
             validate_email(email.strip())
 
+
+def validate_url(url):
```

Comment:
Please add some unit tests for this function and the new class.

Reply:
I added tests of the validate_url.  Curiously, the "with" statement seems to only apply to one "suite" statement, so I had to repeat the "with" clause for each test.

I haven't figured out how to test the MultiFormField yet.   I haven't found an example, and it seems there is a fair amount of context that is required to test properly.